src/test.js

```diff
@@ -2,14 +2,22 @@
 
 const Role = require('../src/role.js');
 const Player = require('../src/player.js');
+const Game = require('../src/game.js');
 let role = new Role();
 let player = new Player();
 console.log(Role.colors);
 
 let players = ['1', '2', '3', '4', '5'];
+let creator = `me`;
+		// 07/16/2021 7:51:47 PM
+		let date = new Date(Date.now()).toLocaleDateString("en-US");
+		let time = new Date(Date.now()).toLocaleTimeString("en-US");
+		let createdOn = `${date} ${time}`;
```

Comment:
Do you know what's up with the formatting? I think this happened on one of my PR's too and I forgot to mention it.